crest/Assets/Crest/Crest/Scripts/Underwater/UnderwaterRenderer.Effect.cs

```diff
@@ -115,8 +115,30 @@ int dataSliceOffset
             Material underwaterPostProcessMaterial = underwaterPostProcessMaterialWrapper.material;
             if (copyParamsFromOceanMaterial)
```

Comment:
just to check, will things still work ok if `copyParamsFromOceanMaterial` is false - i guess it will take params from the global water material as usual?

Reply:
It will take global OR material or a WB material if it finds one which means it could get stuck with WB material. I've added validation to recommend users enable it if they use a WB material. LMKWYT

Reply:
@huwb ping

Reply:
Maybe it should always copy params each frame if WBs with material overrides are present. Or even better, cache which material it last copied params from and only copy when necessary (if "always copy" is not enabled, if it is just always copy). What do you think?

In any case I'm happy for this to merge.

Reply:
The latter sounds great. I've gone with that. Thanks!